src/pages/Mypage/UserInfo/UserInfo.js

```diff
@@ -1,13 +1,57 @@
 import React from 'react';
 import { MYPAGE_CATEGORY } from '../MYPAGE_CATEGORY';
+import { Avatar, Card, CardMedia } from '@mui/material';
+import useFetch from '../../../hooks/useFetch';
 import * as S from './UserInfo.style';
 
 const UserInfo = ({ id }) => {
   const title = MYPAGE_CATEGORY[id].title;
+  const url = '/data/mypageBid.json';
+  const { loading, data } = useFetch(url);
 
   return (
     <S.CategoryContainer>
       <S.Title>{title}</S.Title>
+      <S.UserInfoBox>
+        <S.UserInfoDetail>
+          <Avatar
+            alt="user name"
+            sx={{ width: 70, height: 70, marginRight: 5 }}
+          >
+            정인
+          </Avatar>
+          <S.UserInfoDetailBox>
+            <S.UserName>유정인</S.UserName>
+            <S.UserId>jaylenyu96@gmail.com</S.UserId>
+          </S.UserInfoDetailBox>
+        </S.UserInfoDetail>
+      </S.UserInfoBox>
+      <S.Wishlist>
+        <S.WishlistTitle>위시리스트</S.WishlistTitle>
+        <S.WishlistContainer>
+          {!loading &&
+            data.map((data, id) => (
```

Comment:
```suggestion
            data.map((data, id) => (
```
개발하면서 사용되는 모든것이 data이고 value입니다.

어떤 data인지, parameter에는 어떤 data가 오는지 명확히 알 수 있게 적어주세요